pkg/tool/helmclient/helmclient.go

```diff
@@ -493,6 +538,141 @@ func (hClient *HelmClient) InstallOrUpgradeChart(ctx context.Context, spec *hc.C
 	}
 }
 
+func (hClient *HelmClient) UpdateChartRepo(repoEntry *repo.Entry) (string, error) {
+	chartRepo, err := repo.NewChartRepository(repoEntry, hClient.Providers)
+	if err != nil {
+		return "", err
+	}
+	chartRepo.CachePath = hClient.Settings.RepositoryCache
+
+	// export envionment-variables for ali acr chart repo
+	os.Setenv("HELM_REPO_USERNAME", repoEntry.Username)
```

Comment:
Are these two environment variables officially Helm or Ali? 
Will these environment variables be contaminated if different Helm repoes are executed at the same time?

Reply:
1. Ali acr requests thest two environments explicitly, it won't influence other types of chart repos
2. yes, with very low probability